priv/update-swagger-url

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env escript
+-module('update-swagger-url').
```

Comment:
I don't know about good practices for naming escripts, but I usually do it by replacing `.erl` with `.escript`, so this'd be `update_swagger_url.escript`, and then we could drop the enclosing quotes here (?)